src/types/suggestions.ts

```
@@ -0,0 +1,16 @@
+export interface ClubSuggestion {
+    id: string;
```

ESLint

[error] Delete `··`

(prettier/prettier)